ui/v2.5/src/models/list-filter/criteria/is-missing.ts

```diff
@@ -49,7 +49,7 @@ export const ImageIsMissingCriterionOption = new IsMissingCriterionOptionClass(
   "isMissing",
   "imageIsMissing",
   "is_missing",
-  ["title", "galleries", "studio", "performers", "tags"]
+  ["title", "galleries", "studio", "performers", "tags", "url", "date"]
```

Comment:
URL is covered already in the url criterion.

Reply:
Done